www/nginx/src/opnsense/service/templates/OPNsense/Nginx/nginx

```diff
@@ -1,2 +1,2 @@
-nginx_setup="/usr/local/opnsense/scripts/nginx/setup.php"
+nginx_setup="/usr/local/opnsense/scripts/nginx/setup.php; nginx -t -q 2>&1 | logger -t nginx -p user.emerg"
```

Comment:
I agree with @fabianfrz -- we can move this addition to setup.php with a shell_safe() call